Project.toml

```
@@ -59,7 +59,7 @@ DocStringExtensions = "0.9"
 Flux = "0.15, 0.16"
 GeoMakie = "< 0.7.13"
 HTTP = "1.10"
-Insolation = "0.9.5"
+Insolation = "0.9.5, 0.10"
```

```suggestion
Insolation = "0.10.1"
```
(Insolation 0.10.1 and ClimaParams 0.12.1 fix the epoch parameter issue, so let's just set the compat to that)